### PR TITLE
calendar: remove views height differences

### PR DESCRIPTION
**Pull request by PreslavKozovski**

### CODE REVIEW

**Comment by elena-gancheva** [comment]
Could you please provide some info on why is needed this change? :)

**Comment by PreslavKozovski** [comment]
@elena-gancheva Yep, :)
https://www.screencast.com/t/UDAIoDtXm

File: packages/default/scss/calendar/_layout.scss [hunk 1]
```diff
@@ -149,7 +149,7 @@ $calendar-view-gap: 32px !default;
             .k-content {
                 margin: 0;
                 width: 100%;
-                height: 14.167em;
+                height: 15em;
```
Comment (joneff) [inline packages/default/scss/calendar/_layout.scss]: We are substituting one magic number for another. This should be calculated

File: packages/material/scss/calendar/_layout.scss [hunk 2]
```diff
@@ -52,7 +52,7 @@ $calendar-view-gap: 48px !default;
                 6 * $calendar-month-cell-size,
                 $calendar-weekdays-height,
                 -3 * $calendar-meta-cell-size,
-                .5
+                .45
```
Comment (joneff) [inline packages/material/scss/calendar/_layout.scss]: This should be reverted. The previous logic is precise and correct.

Comment (PreslavKozovski) [reply packages/material/scss/calendar/_layout.scss]: I do believe it is neither precise nor correct. 

As you can see in the below video:
https://www.screencast.com/t/Cy8w7P30CwWI

The height of the "k-month" table is 276px. On the other hand, the height of the "k-meta-view" is 240px. This makes me believe that we should add 36px in padding (2*18px), however, we add 2*20px.

**Comment by PreslavKozovski** [comment]
@joneff ,

I changed the height with min-height because I believe this property should be used in this scenario. 

About the calculations of this magic number, if we decide to do calculations, this height/min-height is not needed at all. However, we will have to do calculations similar to those of the Material theme for every view of the Calendar for every theme. 

Please, let me know what you think about this, and how we should proceed.

**Comment by joneff** [comment]
This PR is superseded by #390 and telerik/kendo#9066

**Pull request closed without merging**